src/fancyrag/utils/env.py

```
@@ -0,0 +1,13 @@
+
+from __future__ import annotations
+
+import os
+
+
+def ensure_env(var: str) -> str:
+    """Return the value of ``var`` or exit the process with an error message."""
+    value = os.getenv(var)
+    if value:
+        return value
+    raise SystemExit(f"Missing required environment variable: {var}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)